salt/scripts.py

```diff
@@ -107,11 +107,6 @@ def salt_minion():
     if '' in sys.path:
         sys.path.remove('')
 
-    if salt.utils.is_windows():
-        minion = salt.cli.daemons.Minion()
-        minion.start()
-        return
-
     if '--disable-keepalive' in sys.argv:
```

Comment:
@twangboy I put it back right here: https://github.com/saltstack/salt/pull/33322/commits/d66c9929183965bd593a156fc4a35100ab3ae8d9